modules/openapi-generator/src/main/resources/kotlin-client/data_class_opt_var.mustache

```diff
@@ -10,10 +10,9 @@
     {{/gson}}
     {{#jackson}}
     {{#isDateTime}}
-    @JsonFormat
-    (shape = JsonFormat.Shape.STRING, pattern = "dd-MM-yyyy hh:mm:ss")
+    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "dd-MM-yyyy hh:mm:ss")
```

Comment:
I think here we should use the iso8601 date format

Reply:
Hey @4brunu , thanks for the feedback.  I'm wondering if it would be better to just remove these `@JsonFormat` annotations in favor of a Jackson serializer/deserializer, similar to the other serialization libraries, such as Moshi, https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/resources/kotlin-client/jvm-common/infrastructure/OffsetDateTimeAdapter.kt.mustache.

I think moving to the serializer/deserializer would likely be better as I mentioned, once you have these `@JsonFormat` annotations, I don't think you can change them, so you'd be stuck with  whatever format you specified here, which I agree should be the iso8601 date format.

Maybe @herveDarritchon could also provide some feedback here as he added these annotations with #5236.

Reply:
Looks like a good idea to me 👍

Reply:
Looks like a good idea to me 👍